app/controllers/completed_tasks_controller.rb

```diff
@@ -0,0 +1,29 @@
+class CompletedTasksController < ApplicationController
+  before_action :authenticate_user!
+  def new
+  	@ct = CompletedTask.new
+  end
+  def create
```

Comment:
Assignment Branch Condition size for create is too high. [32.06/15]<br>Use empty lines between defs.